extension.ts

```diff
@@ -0,0 +1,34 @@
+import {workspace, type ExtensionContext} from 'vscode';
+import {LanguageClient} from 'vscode-languageclient/node';
+
+let client: LanguageClient;
+
+export function activate(context: ExtensionContext) {
+	client = new LanguageClient(
+		'witLanguageClient',
+		'WIT Language Client',
+		{
+			command: context.asAbsolutePath('./out/wit-lsp'),
+		},
+		{
+			documentSelector: [{
+				language: 'wit',
+				pattern: '**/*.wit',
+			}],
+			synchronize: {
+				fileEvents: workspace.createFileSystemWatcher('**/*.wit'),
+			},
+		},
+	);
+
+	// eslint-disable-next-line @typescript-eslint/no-floating-promises
+	client.start();
+}
```

Comment:
Use an environment variable, for now, to change the log level.
```suggestion
import {window, workspace, type ExtensionContext} from 'vscode';
import {LanguageClient} from 'vscode-languageclient/node';

let client: LanguageClient;

export async function activate(context: ExtensionContext): Promise<void> {
	const traceOutputChannel = window.createOutputChannel('WIT Language Server trace');
	client = new LanguageClient(
		'witLanguageClient',
		'WIT Language Client',
		{
			command: context.asAbsolutePath('./out/wit-lsp'),
			options: {
				env: {
					...process.env,
					// eslint-disable-next-line @typescript-eslint/naming-convention
					RUST_LOG: 'debug',
				},
			},
		},
		{
			documentSelector: [{
				scheme: 'file',
				language: 'wit',
			}],
			synchronize: {
				fileEvents: workspace.createFileSystemWatcher('**/*.wit'),
			},
			traceOutputChannel,
		},
	);

	await client.start();
}
```